classes/Mail.php

```diff
@@ -315,7 +315,9 @@ public static function send(
             }
             foreach ($isoArray as $isoCode) {
                 $isoTemplate = $isoCode.'/'.$template;
-                $templatePath = self::getTemplateBasePath($isoTemplate, $moduleName, $shop->theme);
+                if (!empty($templatePath)) {
+                    $templatePath = self::getTemplateBasePath($isoTemplate, $moduleName, $shop->theme);
```

Comment:
Hey, I just realized but `$templatePath` isn't override on each loop. What happened if we don't defined `$templatePath` and we have many values in `$isoArray`?

Reply:
You're right, let me take care about that. I'll come back with the solution

Reply:
So this is not really QA approved? /c @PierreRambaud

Reply:
True, forgot to remove the tag, my bad :/

Reply:
You can do something like:

```php
$newPath = $templatePath;
if (empty($newPath)) {
    $newPath = self::getTemplateBasePath($isoTemplate, $moduleName, $shop->theme);
}
```

And use `$newPath` instead of `$templatePath` (but use a better name like me for variable :D)